trifid/redirect.js

```diff
@@ -54,7 +55,19 @@ function factory () {
         }
       }
       if (shapePath) {
-        return res.redirect(`https://raw.githubusercontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
+        return fetch(`https://raw.githubusercdontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
```

Comment:
There is a typo in the URL:

```suggestion
        return fetch(`https://raw.githubusercontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
```